spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-appactive/src/main/java/com/alibaba/cloud/appactive/consumer/ConsumerAutoConfig.java

```diff
@@ -67,4 +67,16 @@ public RequestInterceptor feignRouterIdTransmissionRequestInterceptor() {
 		return new FeignRouterIdTransmissionRequestInterceptor();
 	}
 
+	@Bean
+	@ConditionalOnMissingBean(name = "restTemplateStrategyBeanPostProcessor ")
+	public BeanPostProcessor restTemplateStrategyBeanPostProcessor() {
+		return new RestTemplateStrategyBeanPostProcessor(context);
+	}
+
+	@Bean
+	@ConditionalOnMissingBean(name = "reactiveRequestStrategyBeanPostProcessor ")
```

Comment:
There is a blank in the string.

Reply:
> There is a blank in the string.

I removed the spaces in the code, please check again, thanks!